avanza/avanza.py

```diff
@@ -1705,3 +1715,108 @@ def create_monthly_saving(
                 }
             }
         )
+
+    def get_transactions(self,
+                         account_or_transaction_type: str,
```

Comment:
We have an enum for transaction_types in constants.py
Can both an account id and a transaction type be present at once?

In that case it would be better to split this argument into two, and add them optionally and validate that at least one of an account id or transaction type is present.

Reply:
It seems like any combination is valid. I.e. one can provide none, one or two of account_id and transaction_type.
I have split the parameter into two separate, optional parameters.